CRM/MembershipExtras/Form/PaymentPlanSettings.php

```diff
@@ -52,6 +70,15 @@ public function buildQuickForm() {
     $this->assign('settingFields', $settingFieldNames);
   }
 
+  /**
+   * Builds an array mapping manual payment processor id's to processor name.
+   *
+   * @return array
+   */
+  private function getManualPaymentProcessors() {
+    return CRM_MembershipExtras_Service_ManualPaymentProcessors::getProcessorsIDNameMap();
```

Comment:
why to wrap this into a new method where u can just call the method directly above, since you are not doing any thing else other than calling getProcessorsIDNameMap() method.